cypress/integration/invoices/invoices.manuallyCreateLineFromPol.spec.js

```diff
@@ -0,0 +1,66 @@
+import TopMenu from '../../support/fragments/topMenu';
+import NewInvoice from '../../support/fragments/invoices/newInvoice';
+import NewInvoiceLine from '../../support/fragments/invoices/newInvoiceLine';
+import Invoices from '../../support/fragments/invoices/invoices';
+import testType from '../../support/dictionary/testTypes';
+import VendorAddress from '../../support/fragments/invoices/vendorAddress';
+import newOrder from '../../support/fragments/orders/newOrder';
+import newOrderLine from '../../support/fragments/orders/newOrderLine';
+import Orders from '../../support/fragments/orders/orders';
+
+describe('ui-invoices: Invoice Line creation - based on POL', () => {
+  const invoice = { ...NewInvoice.defaultUiInvoice };
+  const vendorPrimaryAddress = { ...VendorAddress.vendorAddress };
+  const invoiceLine = { ...NewInvoiceLine.defaultUiInvoiceLine };
+  const order = { ...newOrder.defaultOrder };
+  const orderLine = { ...newOrderLine.defaultOrderLine };
+  const locationName = 'Main Library';
+  const euroCurrency = 'Euro (EUR)';
+  const euroSign = '€';
+
+  before(() => {
+    cy.getToken(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.getOrganizationApi({ query: `name=${invoice.vendorName}` })
+      .then(({ body }) => {
+        invoice.accountingCode = body.organizations[0].erpCode;
+        Object.assign(vendorPrimaryAddress,
+          body.organizations[0].addresses.find(address => address.isPrimary === true));
+        order.vendor = body.organizations[0].id;
+        orderLine.physical.materialSupplier = body.organizations[0].id;
+        orderLine.eresource.accessProvider = body.organizations[0].id;
+      });
+    cy.getBatchGroups()
+      .then(({ body }) => {
```

Comment:
({}) - it's really excess symols, isn't it? Try to write .then(materialTypeId => orderLine.physical.materialType = materialTypeId);